test/functional/ui/float_spec.lua

```diff
@@ -8434,6 +8434,107 @@ describe('float window', function()
                                                   |
         ]]}
       end
+      command('fclose')
+      exec_lua([[
+        local win1, win3 = ...
+        vim.api.nvim_win_set_config(win1, { zindex = 400, title = 'win_400', title_pos = 'center', border = 'single' })
+        vim.api.nvim_win_set_config(win3, { zindex = 300, title = 'win_300', title_pos = 'center', border = 'single' })
```

Comment:
Using different border styles may make the order clearer.

Reply:
okay done.